governance/api-reviews.md

```diff
@@ -200,3 +200,61 @@ When adding new ops, look for:
    it CPU, GPU, XLA, TPU, etc) and prefer to have at least a plan for writing
    device-agnostic code
  - should the python layer for this operation support raggedtensor/sparsetensor?
+
+## Experimental APIs
+
+Experimental APIs are APIs which have the word 'experimental' somewhere in their
+name; for example `tf.experimental.foo`, or `tf.foo.experimental.Bar`, or
+`tf.foo(experimental_bar=True)` or `tf.Foo().experimental_bar()`. We generally
+prefer experimental namespaces when possible, so prefer
+`tf.foo.experimental.Bar` over `tf.foo.ExperimentalBar`.
+
+Experimental APIs are APIs intended to be added to TensorFlow as-is, but which
```

Comment:
Not sure if it would make sense to mention tf/addons and for external users

Reply:
how so?

Reply:
Please correct me if I am wrong: both `experimental` and `tensorflow/addons` are the way to extend the functionality of TensorFlow. 

And I would agree that `experimental` is mostly internal and `addons` external. That's why I was wondering if it would make sense to mention it here.

Reply:
I agree with @lc0 we have also `experimental` in Keras the currently still in the mentioned pattern of `tf.(foo|keras).experimental` namespace and that was not regulated by the approved Keras "separation/migration" RFC approved at https://github.com/tensorflow/community/pull/202.
See also https://github.com/tensorflow/addons/issues/1226#issuecomment-609032397 and the next posts in that thread. /cc @seanpmorgan @tanzhenyu @gabrieldemarmiesse

Reply:
I don't think I want to mention addons here because it's a separate namespace with its own policies.